MicrosoftFluentUI.podspec

```diff
@@ -257,7 +257,6 @@ Pod::Spec.new do |s|
     utilities_ios.source_files = ["ios/FluentUI/Utilities/**/*.{swift,h}"]
   end
 
-
```

Comment:
nit: can you put this line back? we have 2 new lines to separate the mac and ios sections